parachain/runtime/interlay/src/lib.rs

```diff
@@ -322,6 +322,7 @@ impl pallet_aura::Config for Runtime {
     type AuthorityId = AuraId;
     type DisabledValidators = ();
     type MaxAuthorities = MaxAuthorities;
+    type AllowMultipleBlocksPerSlot = ConstBool<false>;
```

Comment:
I am assuming this is related to the async backing feature? Any idea if / when we should set this to true?

Reply:
Described in an inline comment [here](https://github1s.com/paritytech/substrate/blob/HEAD/frame/aura/src/lib.rs#L285-L287). For us, it should be false based on [this comment](https://github1s.com/paritytech/substrate/blob/HEAD/frame/aura/src/lib.rs#L111-L114)